docs/charts.md

```diff
@@ -487,8 +486,7 @@ the Kubernetes objects from the charts and all its dependencies are
 
 Hence a single release is created with all the objects for the chart and its dependencies.
 
-The install order of Kubernetes types is given by the enumeration InstallOrder in kind_sorter.go
-(see [the Helm source file](https://github.com/kubernetes/helm/blob/master/pkg/tiller/kind_sorter.go#L26)).
+The install order of Kubernetes types is given by the enumeration InstallOrder in ***TBD***
```

Comment:
you can use the v3 branch url https://github.com/helm/helm/blob/dev-v3/pkg/tiller/kind_sorter.go#L26